src/wp-includes/blocks/search.php

```diff
@@ -8,11 +8,13 @@
 /**
  * Dynamically renders the `core/search` block.
  *
- * @param array $attributes The block attributes.
+ * @param array    $attributes The block attributes.
+ * @param string   $content    The saved content.
+ * @param WP_Block $block      The parsed block.
```

Comment:
Missing `@since` for new parameters

```
@since 6.3.0 Added the `$content` and `$block` parameter.
```

Reply:
These files are auto-generated from the block-library npm package so any change to this has to be done in Gutenberg. Looking through the other block PHP files, we don't seem to be using `@since` notation in any of them though (or at least not consistently)

Reply:
Yeah I think we have to adopt core standards more consistently in the block-library PHP files given that they're not manually updated in core.

Is this something that should still be done for this release though? As it's essentially a code quality improvement.

Reply:
I'm not sure. It probably depends on when it's easier to do so, as historically adding `@since` could be a massive task 😅.

However, it's a docs-only change, so it can be committed at any time during the cycle, so there's several weeks where the `@since` for 6.3.0 could be added.

Is that doable @tellthemachines?